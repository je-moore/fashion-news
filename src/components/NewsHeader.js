import React from 'react'
import { Typography } from '@material-ui/core'

function NewsHeader({ article }) {
  return (
    <Typography color="textPrimary" variant="h1" align="center">
      Fashion News
    </Typography>
  )
}

export default NewsHeader
