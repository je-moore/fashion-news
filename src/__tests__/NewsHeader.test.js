import React from 'react'
import EnzymeToJson from 'enzyme-to-json'
import { mount } from 'enzyme'
import NewsHeader from '../components/NewsHeader'

it('renders correctly', () => {
  const subject = mount(<NewsHeader />)
  expect(EnzymeToJson(subject)).toMatchSnapshot()
})
