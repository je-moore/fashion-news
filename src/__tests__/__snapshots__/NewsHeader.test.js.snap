// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<NewsHeader>
  <WithStyles(ForwardRef(Typography))
    align="center"
    color="textPrimary"
    variant="h1"
  >
    <ForwardRef(Typography)
      align="center"
      classes={
        Object {
          "alignCenter": "MuiTypography-alignCenter",
          "alignJustify": "MuiTypography-alignJustify",
          "alignLeft": "MuiTypography-alignLeft",
          "alignRight": "MuiTypography-alignRight",
          "body1": "MuiTypography-body1",
          "body2": "MuiTypography-body2",
          "button": "MuiTypography-button",
          "caption": "MuiTypography-caption",
          "colorError": "MuiTypography-colorError",
          "colorInherit": "MuiTypography-colorInherit",
          "colorPrimary": "MuiTypography-colorPrimary",
          "colorSecondary": "MuiTypography-colorSecondary",
          "colorTextPrimary": "MuiTypography-colorTextPrimary",
          "colorTextSecondary": "MuiTypography-colorTextSecondary",
          "displayBlock": "MuiTypography-displayBlock",
          "displayInline": "MuiTypography-displayInline",
          "gutterBottom": "MuiTypography-gutterBottom",
          "h1": "MuiTypography-h1",
          "h2": "MuiTypography-h2",
          "h3": "MuiTypography-h3",
          "h4": "MuiTypography-h4",
          "h5": "MuiTypography-h5",
          "h6": "MuiTypography-h6",
          "noWrap": "MuiTypography-noWrap",
          "overline": "MuiTypography-overline",
          "paragraph": "MuiTypography-paragraph",
          "root": "MuiTypography-root",
          "srOnly": "MuiTypography-srOnly",
          "subtitle1": "MuiTypography-subtitle1",
          "subtitle2": "MuiTypography-subtitle2",
        }
      }
      color="textPrimary"
      variant="h1"
    >
      <h1
        className="MuiTypography-root MuiTypography-h1 MuiTypography-colorTextPrimary MuiTypography-alignCenter"
      >
        Fashion News
      </h1>
    </ForwardRef(Typography)>
  </WithStyles(ForwardRef(Typography))>
</NewsHeader>
`;
